docs/scanners.md

```diff
@@ -23,6 +23,7 @@ Scanners:
 
 10000   Password Autocomplete in browser [Deprecated]
 10001   Secure page browser cache [Deprecated]
+10003   Retire JS
```

Comment:
Just realized I skipped 2 (10002), want me to re-PR?

Reply:
Leave it for next one? :)

Reply:
k